lab-dan/lib/fileRead.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = {};
+
+const fs = require('fs');
+
+module.exports.splice = function (pathArray, printcb, next) {
```

Comment:
Given this is the only function in the module, no need for the named property. Just assign the function directly to module.exports: `module.exports = function()...`